alacritty_config_derive/src/de_struct.rs

```diff
@@ -147,6 +147,18 @@ fn field_deserializer(field_streams: &mut FieldStreams, field: &Field) -> Result
                     log::warn!(target: #LOG_TARGET, #message);
                 });
             },
+            "removed" => {
+                // Construct removal message and append optional attribute override.
+                let mut message = format!("Config warning: {} was removed", literal);
+                if let Some(warning) = parsed.param {
+                    message = format!("{}; {}", message, warning.value());
+                }
+
+                // Append stream to log removal warning.
+                match_assignment_stream.extend(quote! {
+                    log::warn!(target: #LOG_TARGET, #message);
+                });
+            },
```

Comment:
Duplicating this entire block just to separate between is/was seems pointless, why not just branch again?

Reply:
Done.